### ISSUE TITLE
Error while getting Lua linter version

### ISSUE THREAD

**Issue by ferrarimarco** [github/super-linter#602]
**Describe the bug**
Lua linter version is not reported correctly.

**To Reproduce**

```shell
docker run \
    -v "$(pwd)":/workspace \
    -w="/workspace" \
    -e ACTIONS_RUNNER_DEBUG=true \
    -e DEFAULT_WORKSPACE=/workspace \
    -e MULTI_STATUS=false \
    -e RUN_LOCAL=true \
    github/super-linter:V3.8.0

```
**Expected behavior**

**Screenshots**
Verbose output (seen in super-linter v3.8.0):

```shell
[lua]: lua: unrecognized option '--version' usage: lua [options] [script [args]] Available options are:   -e stat  execute string 'stat'   -i       enter interactive mode after executing 'script'   -l name  require library 'name' into global 'name'   -v       show version information   -E       ignore environment variables   --       stop handling options   -        stop handling options and execute stdin
```

**Additional context**
This is not the first time this happens, see #543.  I highly suggest throwing an error and terminating the script, because if you cannot be sure of the linter version you're using, you might get inconsistent results.
